.github/workflows/update-desktop-mirror.yml

```
@@ -0,0 +1,20 @@
+name: Build
+on:
+  push:
+    branches: [ 'trunk' ]
+
+jobs:
+  update_mirrors:
+    name: Push to desktop mirror
+    runs-on: ubuntu-latest
+
+    if: github.event_name == 'push' && github.repository == 'Automattic/wp-calypso'
+
+    steps:
+      - name: Push to wp-desktop repository
```

I think you need a step checking out the repo before trying to push here?

```suggestion
      - name: Checkout repo
        uses: actions/checkout@v2
      - name: Push to wp-desktop repository
```